lms/djangoapps/labster_vouchers/views.py

```diff
@@ -178,7 +178,13 @@ def activate_voucher_view(request):
         messages.error(request, _(u"No course found for enrollment."))
         return redirect(enter_voucher_url)
     except CourseEnrollmentExistsError:
-        messages.error(request, _(u"You have been already enrolled to the course before."))
+        messages.error(
+            request,
+            _(
+                u"You have successfully activated the course code and enrolled yourself in the course. "
```

Comment:
@lukyfebrianto it can be either course code or voucher. Is it okay to use `course code` word for both cases?

Reply:
Please use `access code` instead. See examples above.

Reply:
Yes it make sense. we also use voucher in this form. 
changed